### ISSUE TITLE
Exporting to edgeTPU doesn't work

### ISSUE THREAD

**Issue by csongorkeller** [ultralytics/ultralytics#14692]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Export

### Bug

I've tried in many environments, even with your managed Google Colab setup and exporting to edgeTpu format doesn't work. It just crashes google colab after a few minutes

These are the logs I get:
Ultralytics YOLOv8.2.65 🚀 Python-3.10.12 torch-2.3.1+cu121 CPU (Intel Xeon 2.00GHz)
WARNING ⚠️ INT8 export requires a missing 'data' arg for calibration. Using default 'data=coco8.yaml'.
YOLOv8x summary (fused): 268 layers, 68,200,608 parameters, 0 gradients, 257.8 GFLOPs

PyTorch: starting from 'yolov8x.pt' with input shape (1, 3, 640, 640) BCHW and output shape(s) (1, 84, 8400) (130.5 MB)

TensorFlow SavedModel: starting export with tensorflow 2.16.1...

ONNX: starting export with onnx 1.16.1 opset 17...
ONNX: slimming with onnxslim 0.1.32...
ONNX: export success ✅ 10.4s, saved as 'yolov8x.onnx' (260.5 MB)
TensorFlow SavedModel: collecting INT8 calibration images from 'data=coco8.yaml'
Scanning /content/datasets/coco8/labels/val.cache... 4 images, 0 backgrounds, 0 corrupt: 100%|██████████| 4/4 [00:00<?, ?it/s]TensorFlow SavedModel: WARNING ⚠️ >300 images recommended for INT8 calibration, found 4 images.
TensorFlow SavedModel: starting TFLite export with onnx2tf 1.22.3...

Automatic generation of each OP name started ========================================

Automatic generation of each OP name complete!

Model loaded ========================================================================

Model conversion started ============================================================
saved_model output started ==========================================================

### Environment

https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb

### Minimal Reproducible Example

from ultralytics import YOLO

model = YOLO('yolov8x.pt') 

model.export(format='edgetpu',int8=True)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by hasanghaffari93** [comment]
Hi @csongorkeller 

Exporting the largest YOLOv8 (`yolov8x`) needs more resources (RAM) which crashes Google Colab with 12 GB System RAM (see below). You can try a smaller model (like `yolov8n`) which is more optimized in terms of speed in USB accelerator.

![Screenshot from 2024-07-25 21-26-17](https://github.com/user-attachments/assets/af25bbfe-7f3e-48dd-8fb1-b745a7aec674)

**Comment by csongorkeller** [comment]
Hi @hasanghaffari93 

Thanks a lot for the info. With the smaller models indeed the exporting works for Google Colab. Will try exporting yolov8x on a bigger beast.
